fdbrpc/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ if(NOT WIN32)
   list(APPEND FDBRPC_THIRD_PARTY_SRCS libcoroutine/context.c libeio/eio.c)
 endif()
 
-add_library(thirdparty ${FDBRPC_THIRD_PARTY_SRCS})
+add_library(thirdparty STATIC ${FDBRPC_THIRD_PARTY_SRCS})
```

Comment:
I think this change is ok, but FWIW I don't see any cyclic dependencies involving `thirdparty`

Reply:
There weren't, I just went manually looking at lib/ to figure out what was going on, and saw that this was being built as a .so for no reason.